Add vsftp and JanusGraph files (#367)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/vsftpd.md

---
name: vsftpd
category: Networking

> description: vsftpd is a secure, high-performance FTP server for Unix-like systems, designed for reliability and strong security features.

Author: ```suggestion
description: Vsftpd is a secure, high-performance FTP server for Unix-like systems, designed for reliability and strong security features.
```

download_url: https://security.appspot.com/vsftpd.html#download
works_on_arm: true
supported_minimum_version:
    version_number: 3.0.3.1
    release_date: 2016/05/25
 
 
optional_info:
    homepage_url: https://security.appspot.com/vsftpd.html
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:
        partner_content:
        official_docs: https://github.com/djarosz/vsftpd/blob/master/INSTALL
    arm_recommended_minimum_version:
        version_number:
        release_date:
 
optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info: Release notes for Linux/Arm64 is not avialable. v3.0.3.1 has been successfully installed on the Neoverse N1.

Author: ```suggestion
    other_info: Linux/ARM64 release notes are not available. v3.0.3.1 has been successfully installed on the Neoverse N1.
```
